File: SDKs/Aspose.Pdf_Cloud_SDK_for_Java/src/main/java/com/aspose/pdf/model/WordCount.java
package com.aspose.pdf.model;

import java.util.*;
public class WordCount {
  private List<PageWordCount> List = new ArrayList<PageWordCount>();
  /**
	 * getList
	 * Gets List<PageWordCount>
	 * @return List
	 */
  public List<PageWordCount> getList() {
    return List;
  }

	/**
	 * setList
	 * Sets List<PageWordCount>
	 * @param List List<PageWordCount>
	 */
  public void setList(List<PageWordCount> List) {
    this.List = List;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class WordCount {\n");
    sb.append("  List: ").append(List).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Pdf-Cloud-SDK-for-.NET/src/Com/Aspose/PDF/Model/WordCount.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class WordCount {
    public List<PageWordCount> List { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class WordCount {\n");
      sb.Append("  List: ").Append(List).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
